examples_gbs/run_tutorial_dense.py

```diff
@@ -0,0 +1,120 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Dense Subgraph Tutorial
+=========================
+Graphs can be used to model a wide variety of objects: social networks, financial markets,
+biological networks, and many others. A common problem of interest is to find subgraphs that
+contain a large number of connections between its nodes. These may correspond to communities in
+social networks, correlated assets in a market, or mutually influential proteins in a biological
+network. This tutorial studies how GBS can be used to identify dense subgraphs.
+
+Mathematically, this task is known as the `dense subgraph problem
+<https://en.wikipedia.org/wiki/Dense_subgraph>`_. The density of a graph is equal to the number of
+edges in the subgraph divided by the maximum possible number of edges. Identifying the densest

```

Comment:
How about "The density of a :math:`k`-node subgraph is equal to the number of its edges divided by the maximum possible number of edges :math:`n(n-1)/2`.